Development/src/pages/menu.js

```diff
@@ -24,118 +23,103 @@ import SettingsInputAntennaIcon from '@material-ui/icons/SettingsInputAntenna';
+                            </ListItemIcon>
+                            <ListItemText primary="General" />
+                        </StyledListItem>
+                        <StyledListItem
+                            button
+                            component={NavLink}
+                            to={'/queryapis'}
+                        >
+                            <ListItemIcon>
+                                <QueryIcon />
+                            </ListItemIcon>
+                            <ListItemText
+                                style={{ whiteSpace: 'nowrap' }}
+                                primary="Query APIs"
+                            />
+                        </StyledListItem>
+                    </List>
+                </Collapse>
+            </Paper>
+            <StyledListItem button component={NavLink} to={'/nodes'}>
+                <ListItemIcon>
+                    <DvrIcon />
+                </ListItemIcon>
+                <ListItemText primary="Nodes" />
+            </StyledListItem>
+            <StyledListItem button component={NavLink} to={'/devices'}>
+                <ListItemIcon>
+                    <SettingsCellIcon />
+                </ListItemIcon>
+                <ListItemText primary="Devices" />
+            </StyledListItem>
+            <StyledListItem button component={NavLink} to={'/sources'}>
+                <ListItemIcon>
+                    <SettingsInputAntennaIcon />
+                </ListItemIcon>
+                <ListItemText primary="Sources" />
+            </StyledListItem>
+            <StyledListItem button component={NavLink} to={'/flows'}>
+                <ListItemIcon>
+                    <CompareArrowsIcon />
+                </ListItemIcon>
+                <ListItemText primary="Flows" />
+            </StyledListItem>
+            <StyledListItem button component={NavLink} to={'/senders'}>
+                <ListItemIcon>
+                    <CallMadeIcon />
+                </ListItemIcon>
+                <ListItemText primary="Senders" />
+            </StyledListItem>
+            <StyledListItem button component={NavLink} to={'/receivers'}>
+                <ListItemIcon>
+                    <CallReceivedIcon />
+                </ListItemIcon>
+                <ListItemText primary="Receivers" />
+            </StyledListItem>
+            <StyledListItem button component={NavLink} to={'/subscriptions'}>
+                <ListItemIcon>
+                    <SwapVertIcon />
+                </ListItemIcon>
+                <ListItemText primary="Subscriptions" />
+            </StyledListItem>
+            <StyledListItem button component={NavLink} to={'/events'}>
+                <ListItemIcon>
+                    <AnnouncementIcon />
+                </ListItemIcon>
+                <ListItemText primary="Logs" />
+            </StyledListItem>
+        </MenuList>
     );
 };
 
-NestedList.propTypes = {
```

Comment:
This has been removed as it was an outlier however it would be beneficial to check prop types on all components